### DISCUSSION TITLE
Clarifications on settings

### DISCUSSION THREAD

**Discussion by lkaino** [xtremestuff/protune-transforms#6] [Q&A]
I tried both methods, using the ACES transforms and the LUT method using .dctl files.

First, the LUT method.

I used following settings, are these correct? I don't know what is the difference between Color Managed or not.
![image](https://github.com/xtremestuff/protune-transforms/assets/6444272/9eae5408-2a12-40ba-b261-24df98faaa77)

I dragged the LUT GoPro_Protune_Native_Rec709_Gamma-2.2_v2.dctl to a clip shot in Flat Native.

When I render the clip with the LUT, I get color banding as can be seen in this image:
![image](https://github.com/xtremestuff/protune-transforms/assets/6444272/ecc45ce8-128b-4d84-8a74-2b1ff3560334)

Is this banding caused by the free version of resolve, do I need Studio to remove them?

Next I tried the ACES with following settings:
![image](https://github.com/xtremestuff/protune-transforms/assets/6444272/9aa26682-bca9-4d6a-a8be-d190a7d231ea)

Are these settings correct?

The video renders without banding and looks good.

I would prefer the non-ACES way, as the grading seems easier to with it. With ACES for example the "lift" wheel transforms the whole color space, not only the bottom section.

**Comment by lkaino** [comment]
I also tried these settings, but the color banding remains in the rendered video
![image](https://github.com/xtremestuff/protune-transforms/assets/6444272/fa4fea84-90f4-4337-b1fc-f4533f569a2a)

**Reply by RikshaDriver** [reply]
If the banding is not present in the ACES implementation, then it could possibly be due to the non-studio watermark, but I can't say for certain.

Personally, I would say go down the ACES path and tick "Color space aware grading tools".

RCMv2 has far too many hidden gotchas.
